README.md

```diff
@@ -72,7 +72,7 @@ In addition, we also provide a Python frontend for [PennyLane](https://pennylane
 
 ## Installation
 
-Catalyst is officially supported on Linux (x86_64) and macOS (aarch64) platforms, and pre-built
+Catalyst is officially supported on Linux and macOS, on both x86_64 and aarch64 platforms, and pre-built

```

Comment:
Just confirming, there should be no difference in the installation command for the shipped wheels for aarch and x86 right? aka the pip install line later in the installation doesn't need to be changed.

And on that topic, what about building from source guide? Anything needed there?

Reply:
I have re-ran the installation from source in the Mac1 machine and with manylunux_2_28_aarch64 and I can confirm that `make all` works correctly